master/internal/resourcemanagers/kubernetes_resource_manager.go

```diff
@@ -133,7 +136,10 @@ func (k *kubernetesResourceManager) Receive(ctx *actor.Context) error {
 		ctx.Respond(getTaskSummaries(k.reqList, k.groups, kubernetesScheduler))
 
 	case *apiv1.GetResourcePoolsRequest:
-		resourcePoolSummary := k.summarizeDummyResourcePool(ctx)
+		resourcePoolSummary, err := k.summarizeDummyResourcePool(ctx)
+		if err != nil {
+			return err
```

Comment:
this will crash the resource manager on error - we'd rather not do that. the actor system has lots of pitfalls like this. instead `ctx.Respond(err)`.